### ISSUE TITLE
Feature request: entry point for CLI usage

### ISSUE THREAD

**Issue by tekumara** [mingrammer/diagrams#369]
If the diagrams package had an entry point, it could be used via the cli in an isolated virtual environment, eg:

```
pipx install diagrams
diagrams my-diagram.py
```

**Comment by m1racoli** [comment]
Yes, it would be nice to be able to have a CLI to generate diagrams. With pipx or brew this could then be made available system wide. No need to set up a virtual env.
